trie/test_walker.py

```diff
@@ -0,0 +1,440 @@
+import enum
+from itertools import zip_longest
+from typing import (
+    NamedTuple,
+    Set,
+    Tuple,
+)
+
+from eth_typing import (
+    Hash32,
+)
+from eth_utils import (
+    to_bytes,
+    to_tuple,
+    ValidationError,
+)
+from sortedcontainers import SortedSet
```

Comment:
A brief look at the documentation gives me lots of confidence in this library.  If your experience continues to be positive after further depth with the libary you might add this to the tactical manual with a snippet of your experience.